boards/build_scripts/tock_kernel_layout.ld

```diff
@@ -44,40 +44,44 @@ PAGE_SIZE = DEFINED(PAGE_SIZE) ? PAGE_SIZE : 512;
-     * significant for systems like QEMU that use multiboot. Only the first 8192 bytes of the file
-     * are scanned ( https://www.gnu.org/software/grub/manual/multiboot/multiboot.html ).
+    /*
+     * The order that section contents ends up emitted in the final physical ELF
+     * file is significant for systems like QEMU that use multiboot. Only the
+     * first 8192 bytes of the file are scanned
+     * (https://www.gnu.org/software/grub/manual/multiboot/multiboot.html).
      *
-     * For this reason, it is important that the `.text` section, which contains the `.multiboot`
-     * section, is defined first in this file as tools tend to write sections out in linear order.
+     * For this reason, it is important that the `.text` section, which contains
+     * the `.multiboot` section, is defined first in this file as tools tend to
+     * write sections out in linear order.
      */
-    
-   
+
+
     /* STATIC ELEMENTS FOR TOCK KERNEL */
     .text :
     {
         . = ALIGN(4);
-        _textstart = .;         /* Symbol expected by some MS build toolchains */
+        _textstart = .;     /* Symbol expected by some MS build toolchains */
         _stext = .;         /* First of standard s,e (start/end) pair */
 
-        /* x86     
-         * We need to make sure that qemu-system-i386 finds the multiboot header within the
-         * first 8K of the ELF. The multiboot header is placed at the beginning of the .text section, so
-         * this section has to be placed first in the file.
+        /* x86
+         * We need to make sure that qemu-system-i386 finds the multiboot header
+         * within the first 8K of the ELF. The multiboot header is placed at the
+         * beginning of the .text section, so this section has to be placed
+         * first in the file.
          *
          * The multiboot header's documentation is available at
          * https://www.gnu.org/software/grub/manual/multiboot/multiboot.html.
-         * This allows Tock to be booted using a bootloader that supports multiboot v1.
-         * QEMU provides for x86 such a bootloader when used with the `-kernel` flag.
+         * This allows Tock to be booted using a bootloader that supports
+         * multiboot v1. QEMU provides for x86 such a bootloader when used with
+         * the `-kernel` flag.
          *
-         * The bootloader is responsable for loading the kernel into RAM and executing it. Without it,
-         * the Tock kernel should provide a bootloader that starts reading the kernel into RAM.
+         * The bootloader is responsible for loading the kernel into RAM and
+         * executing it. Without it, the Tock kernel should provide a bootloader
+         * that starts reading the kernel into RAM.
          *
          * This is the reason why the .text section has to be the first.
          */
-        KEEP(*(.multiboot))
+        KEEP (*(.multiboot))
```

Comment:
I see that this is inconsistent in this file, but the other one should change. I.e., it should be `KEEP()` (no space) everywhere. That's in line with `ld`'s "directives" conceptually being like function calls, and [matches the spacing used in `ld`'s documentation](https://sourceware.org/binutils/docs/ld.html).